rs/nervous_system/integration_tests/tests/advance_target_version.rs

```diff
@@ -0,0 +1,141 @@
+use ic_nervous_system_integration_tests::pocket_ic_helpers::sns;
+use ic_nervous_system_integration_tests::{
+    create_service_nervous_system_builder::CreateServiceNervousSystemBuilder,
+    pocket_ic_helpers::{
+        add_wasm_via_nns_proposal, add_wasms_to_sns_wasm, install_nns_canisters, nns,
+    },
+};
+use ic_nns_test_utils::sns_wasm::create_modified_sns_wasm;
+use ic_sns_governance::pb::v1 as sns_pb;
+use ic_sns_swap::pb::v1::Lifecycle;
+use ic_sns_wasm::pb::v1::SnsCanisterType;
+use pocket_ic::PocketIcBuilder;
+use std::time::Duration;
+
+const TICKS_PER_TASK: u64 = 2;
+
+#[test]
+fn test_get_upgrade_journal() {
+    let pocket_ic = PocketIcBuilder::new()
+        .with_nns_subnet()
+        .with_sns_subnet()
+        .build();
+
+    let wait_for_next_periodic_task = |sleep_duration_seconds| {
+        let now = pocket_ic.get_time();
+        pocket_ic.advance_time(Duration::from_secs(sleep_duration_seconds));
+        for _ in 0..TICKS_PER_TASK {
+            pocket_ic.tick();
+        }
+        assert_eq!(
+            pocket_ic.get_time(),
+            now + Duration::from_secs(sleep_duration_seconds)
+                + Duration::from_nanos(TICKS_PER_TASK)
+        );
+    };
+
+    // Install the (mainnet) NNS canisters.
+    let with_mainnet_nns_canisters = false;
+    install_nns_canisters(&pocket_ic, vec![], with_mainnet_nns_canisters, None, vec![]);
+
+    // Publish (mainnet) SNS Wasms to SNS-W.
```

Comment:
comment here and line 37 seems wrong "(mainnet)"

Reply:
Resolved